tests/e2e_logging.py

```diff
@@ -51,6 +51,27 @@ def test_large_messages(network, args):
     return network
 
 
+@reqs.description("Write/Read with cert prefix")
+@reqs.supports_methods("LOG_record_prefix_cert", "LOG_get")
+def test_cert_prefix(network, args):
+    if args.package == "liblogging":
```

Comment:
We've already got a `lua_generic_app` decorator. I wonder if we should just add one for our C++ and JS apps too?

Reply:
We can add `reqs.cpp_logging_app` and `reqs.js_generic_app`, but we're covering the precise requirement here with `supports_method`, and I believe we still need this guard because we're not actually enforcing the decorators? (That's what `test_update_lua` is doing)

Reply:
You're right - decorators are not enforced in individual tests.